lib/iris/fileformats/pp.py

```diff
@@ -673,8 +673,8 @@ def __init__(self, value):
 
         """
         if value < 0:
-            raise ValueError('Negative numbers not supported with '
-                             'splittable integers object')
+            msg = 'Setting LBPROC with a negative value of {}'
```

Comment:
I'm happy either way. In general I'd lean towards no warning, but given the meaning of negative values is not defined in to the UM docs, I'm ok with telling the user.